src/blindrsa/blindrsa.test.ts

```diff
@@ -0,0 +1,95 @@
+import blindrsa from './index';
+import { jest } from '@jest/globals';
+import sjcl from './sjcl';
+// Test vector
+// https://www.ietf.org/archive/id/draft-irtf-cfrg-rsa-blind-signatures-03.html#appendix-A
+import vectors from './testdata/rsablind_vectors.json';
+
+function hexToB64URL(x: string): string {
+    return Buffer.from(x, 'hex').toString('base64url');
+}
+
+function hexToUint8(x: string): Uint8Array {
+    return new Uint8Array(Buffer.from(x, 'hex'));
+}
+
+function paramsFromVector(v: typeof vectors[number]): {
+    n: string;
+    e: string;
+    d: string;
+    p: string;
+    q: string;
+    dp: string;
+    dq: string;
+    qi: string;
+} {
+    const n = hexToB64URL(v.n);
+    const e = hexToB64URL(v.e);
+    const d = hexToB64URL(v.d);
+    const p = hexToB64URL(v.p);
+    const q = hexToB64URL(v.q);
+
+    // Calculate CRT values
+    const bnD = new sjcl.bn(v.d);
+    const bnP = new sjcl.bn(v.p);
+    const bnQ = new sjcl.bn(v.q);
+    const one = new sjcl.bn(1);
+    const dp = hexToB64URL(bnD.mod(bnP.sub(one)).toString());
+    const dq = hexToB64URL(bnD.mod(bnQ.sub(one)).toString());
+    const qi = hexToB64URL(bnQ.inverseMod(bnP).toString());
+    return { n, e, d, p, q, dp, dq, qi };
+}
+
+async function keysFromVector(v: typeof vectors[number]): Promise<CryptoKeyPair> {
+    const params = paramsFromVector(v);
+    const { n, e } = params;
+    const publicKey = await crypto.subtle.importKey(
+        'jwk',
+        { kty: 'RSA', ext: true, n, e },
+        { name: 'RSA-PSS', hash: 'SHA-384' },
+        true,
+        ['verify'],
+    );
+
+    const privateKey = await crypto.subtle.importKey(
+        'jwk',
+        { kty: 'RSA', ext: true, ...params },
+        { name: 'RSA-PSS', hash: 'SHA-384' },
+        true,
+        ['sign'],
+    );
+    return { privateKey, publicKey };
+}
+
+describe('BlindRSA', () => {
+    test.each(vectors)('vec$#', async (v: typeof vectors[number]) => {
+        const r_inv = new sjcl.bn(v.inv);
+        const r = r_inv.inverseMod(new sjcl.bn(v.n));
+        const r_bytes = hexToUint8(r.toString().slice(2));
+
+        const { privateKey, publicKey } = await keysFromVector(v);
+        const msg = hexToUint8(v.msg);
+        const saltLength = v.salt.length / 2;
+
+        // Mock for randomized blind operation.
+        jest.spyOn(crypto, 'getRandomValues')
+            .mockReturnValueOnce(hexToUint8(v.salt)) // mock for random salt
+            .mockReturnValueOnce(r_bytes); // mock for random blind
+
+        const { blinded_msg, blinded_inv } = await blindrsa.blind(publicKey, msg, saltLength);
+        expect(blinded_msg).toStrictEqual(hexToUint8(v.blinded_msg));
+        expect(blinded_inv).toStrictEqual(hexToUint8(v.inv));
+
+        const blind_sig = await blindrsa.blind_sign(privateKey, blinded_msg);
```

Comment:
nit: is the naming convention `blind_sign` or `blindSign`?

Reply:
unified naming convention.